<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomManageMainWindow</class>
 <widget class="QMainWindow" name="RoomManageMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>694</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RoomManageMainWindow{
border-image: url(:/mainbackground.jpg);
}
#RoomManageMainWindow *{
border-image: url();
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>7</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>41</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="minimumSize">
           <size>
            <width>629</width>
            <height>271</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableView QTableCornerButton::section {
   /*  background: red;
    border: 2px outset red;*/
    color: red;
    background-color: rgba(129, 207, 209, 30%);
    border: 5px solid #f6f7fa;
    border-radius:1px;
    border-color: rgba(129, 207, 209, 30%);
 }
QTableView {
    color: black;                                       /*表格内文字颜色*/
    gridline-color: black;                              /*表格内框颜色*/
    background-color: rgba(108, 108, 108, 0%);               /*表格内背景色*/
    alternate-background-color: rgb(64, 64, 64);
    selection-color: black;                             /*选中区域的文字颜色*/
    selection-background-color: rgb(195, 213, 223);        /*选中区域的背景色*/
    border: 2px groove gray;
    border-radius: 0px;
    padding: 2px 4px;
}
QHeaderView {
    color: black;
	font: 9pt &quot;微软雅黑&quot;;
    background-color: rgba(195, 213, 223, 0%);/*设置表头空白区域背景色*/
    border: 0px solid rgb(144, 144, 144);
    border:0px solid rgb(191,191,191);
    border-left-color: rgba(255, 255, 255, 0);
    border-top-color: rgba(255, 255, 255, 0);
    border-radius:0px;
    min-height:29px;
}
QHeaderView::section {
    color: black;
    background-color: rgba(129, 207, 209, 30%);
    border: 5px solid #f6f7fa;
    border-radius:1px;
    border-color: rgba(129, 207, 209, 30%);
} </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTableView" name="ordertableView">
          <property name="minimumSize">
           <size>
            <width>629</width>
            <height>251</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableView QTableCornerButton::section {
   /*  background: red;
    border: 2px outset red;*/
    color: red;
    background-color: rgba(129, 207, 209, 30%);
    border: 5px solid #f6f7fa;
    border-radius:1px;
    border-color: rgba(129, 207, 209, 30%);
 }
QTableView {
    color: black;                                       /*表格内文字颜色*/
    gridline-color: black;                              /*表格内框颜色*/
    background-color: rgba(108, 108, 108, 0%);               /*表格内背景色*/
    alternate-background-color: rgb(64, 64, 64);
    selection-color: black;                             /*选中区域的文字颜色*/
    selection-background-color: rgb(195, 213, 223);        /*选中区域的背景色*/
    border: 2px groove gray;
    border-radius: 0px;
    padding: 2px 4px;
}
QHeaderView {
    color: black;
	font: 9pt &quot;微软雅黑&quot;;
    background-color: rgba(195, 213, 223, 0%);/*设置表头空白区域背景色*/
    border: 0px solid rgb(144, 144, 144);
    border:0px solid rgb(191,191,191);
    border-left-color: rgba(255, 255, 255, 0);
    border-top-color: rgba(255, 255, 255, 0);
    border-radius:0px;
    min-height:29px;
}
QHeaderView::section {
    color: black;
    background-color: rgba(129, 207, 209, 30%);
    border: 5px solid #f6f7fa;
    border-radius:1px;
    border-color: rgba(129, 207, 209, 30%);
} </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="queryEmptyRoomButton">
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>29</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>29</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color: rgb(104, 160, 177); 
color: rgb(255, 255, 255); }</string>
          </property>
          <property name="text">
           <string>查询空房</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="orderRoomButton">
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color: rgb(104, 160, 177); 
color: rgb(255, 255, 255); }</string>
          </property>
          <property name="text">
           <string>添加预定信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelupdateButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color:rgb(195, 213, 223); color: rgb(122, 136, 127);}
</string>
          </property>
          <property name="text">
           <string>取消登记的信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="liveinButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color:rgb(195, 213, 223); color: rgb(122, 136, 127);}</string>
          </property>
          <property name="text">
           <string>确认登记信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchOrderEdit">
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 70%);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchOrderButton">
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color: rgb(104, 160, 177); 
color: rgb(255, 255, 255); }</string>
          </property>
          <property name="text">
           <string>搜索该客户房间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="beginupdateButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color:rgb(195, 213, 223); color: rgb(122, 136, 127);}</string>
          </property>
          <property name="text">
           <string>更改信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelchangeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color:rgb(195, 213, 223); color: rgb(122, 136, 127);}</string>
          </property>
          <property name="text">
           <string>取消更改的信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="checkupdateButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color:rgb(195, 213, 223); color: rgb(122, 136, 127);}</string>
          </property>
          <property name="text">
           <string>确认更改信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletcheckinButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color:rgb(195, 213, 223); color: rgb(122, 136, 127);}</string>
          </property>
          <property name="text">
           <string>删除该预定信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="orderToliveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color:rgb(195, 213, 223); color: rgb(122, 136, 127);}</string>
          </property>
          <property name="text">
           <string>确认转入住</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="checkoutButton">
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color: rgb(104, 160, 177); 
color: rgb(255, 255, 255); }</string>
          </property>
          <property name="text">
           <string>退房</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { background-color: rgb(104, 160, 177); 
color: rgb(255, 255, 255); }</string>
          </property>
          <property name="text">
           <string>返回主界面</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>41</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
